sklearn/utils/estimator_checks.py

```diff
@@ -186,6 +189,13 @@ def _yield_transformer_checks(name, Transformer):
         # basic tests
         yield check_transformer_general
         yield check_transformers_unfitted
+    # Dependent on external solvers and hence accessing the iter
+    # param is non-trivial.
+    external_solver = ['Isomap', 'KernelPCA', 'LocallyLinearEmbedding',
```

Comment:
Hm I guess these were skipped manually before, so it's ok to skip them like this for this PR. It's really bad practice but I I think the PR is a step in the right direction.